libs/compute_api/src/responses.rs

```diff
@@ -93,6 +93,15 @@ pub enum TerminateMode {
     Immediate,
 }
 
+impl From<TerminateMode> for ComputeStatus {
```

Comment:
Why do we still need TerminateMode?

Reply:
We need to parse query parameters for `/terminate` route, see changes in the PR